api/swagger-spec/api.json

```diff
@@ -0,0 +1,27 @@
+{
+  "swaggerVersion": "1.2",
+  "apiVersion": "",
+  "basePath": "https://10.192.210.63:8443",
```

Comment:
That breaks startup, because all of the master start code craps the bed if it can't talk to itself.

----- Original Message -----
> > @@ -0,0 +1,27 @@
> > +{
> > +  "swaggerVersion": "1.2",
> > +  "apiVersion": "",
> > +  "basePath": "https://10.192.210.63:8443",
> 
> > Unfortunately, it is.
> 
> If this is generated, you can likely get it to generate correctly by forcing
> `--listen` to "127.0.0.1".
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/2660/files#r31465761